server/vulnerabilities/cpe_test.go

```diff
@@ -102,7 +101,7 @@ func TestSyncCPEDatabase(t *testing.T) {
 	require.Error(t, err)
 
 	// and we make the db older than the release
-	newTime := time.Date(2000, 01, 01, 01, 01, 01, 01, time.UTC)
+	newTime := time.Date(2000, 0o1, 0o1, 0o1, 0o1, 0o1, 0o1, time.UTC)
```

Comment:
Nit, but I don't think the intent was to have it in octal, this makes it look like there's something more complex going on (it's 1 anyway).

Reply:
Good catch, will revert, this was done automatically by some lint tool in VSCode.